src/reducers/walletConnectReducer.js

```diff
@@ -58,34 +119,39 @@ const initialState = {
   waitingRequest: null,
 };
 
-export default function walletConnectReducer(
+const walletConnectReducer = (
   state: WalletConnectReducerState = initialState,
   action: WalletConnectReducerAction,
-) {
-  const { payload } = action;
```

Comment:
This is just because of flow getting confused, I'll look into it on the next update.